org.keystore_explorer.KeyStoreExplorer.yml

```diff
@@ -0,0 +1,53 @@
+app-id: org.keystore_explorer.KeyStoreExplorer
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+command: kse
+rename-desktop-file: kse.desktop
+finish-args:
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=home
```

Comment:
why do you need `home`? Does it support filechooser portals?

Reply:
- `home` is not explicitly needed, Downloads or Documents can work, the app main focus is to work with files (certificates, keys, etc..) so I though `home` is more suitable default. let me know if I should switcht to something else.
- It does not support filechooser portal.

Reply:
If you can tighten it, it's better but certs and keys are often in project directories and not in a standard xdg-downloads directory or similar.

Reply:
switched to `xdg-documents/kse:create`